src/controllers/auth.controller.js

```
@@ -17,23 +18,24 @@ export const createClient = async (req, res) => {
     const newClient = await client.getSafeDataValues();
     const token = generateToken(newClient);
 
-    return successResponse(
-      res,
-      201,
-      'Your account has been created successfully',
-      { [role]: newClient, token }
-    );
+    return successResponse(res, 201, 'Your account has been created successfully', {
+      [role]: newClient,
+      token,
+    });
   } catch (error) {
-    return serverError(res);
+    return serverError(error);
```

Please return a meaningful error message, not the `error `object itself

---

That's not my code. My error message is in line 41. Should I correct it for the person all the same??